porting/nimble/src/nimble_port.c

```diff
@@ -44,14 +38,6 @@ nimble_port_init(void)
 
     ble_hs_init();
 
-    /* XXX These should be configurable somehow */
-    ble_svc_gap_init();
```

Comment:
I'm not sure about gap service. I would keep it here imho as this is needed but not everybody is aware of it.

Reply:
it's mandatory for peripheral and central so should only be included if needed. in Mynewt app needs to explicitly add dep to package for GAP (and also GATT) service to be included so it works pretty much the same here.

Reply:
I agree